config/keys.js

```diff
@@ -1,6 +1,8 @@
+// TODO: remove this comments
+// "mongodb://mernstack:mernstack1@ds247377.mlab.com:47377/devconnector",
+// "mongodb://localhost:27017/devconnector",
+
 module.exports = {
-  mongoURI:
-    // "mongodb://mernstack:mernstack1@ds247377.mlab.com:47377/devconnector",
-    "mongodb://localhost:27017/devconnector",
+  mongoURI: `mongodb://${process.env.DB_URI}/devconnector`,
```

Comment:
This forces the DB_URI env variable for non-containerised runs as well, might be good to document or change the npm script or implement a default.